File: MS.NET/clrzbus/examples/rpc/RpcClient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using zbus;
namespace zbus
{
    
    class RpcClient
    {
        public static void Main(string[] args)
        {
            //1) 配置RPC链接
            RpcConfig config = new RpcConfig();
            config.Service = "MyRpc";  
            Rpc rpc = new Rpc(config);


            //2) 远程方法调用
            object res = rpc.Invoke("user", "前海股权交易中心");  
            IDictionary<string, object> user = (Dictionary<string, object>)res; 
            Console.WriteLine("name: " + user["name"]);
            Console.WriteLine("addr: " + user["addr"]);

            //3) 销毁链接
            rpc.Dispose();
            Console.ReadKey();
        } 
    }
}

File: PYTHON/pyzbus/test/rpc/rpc_client.py
# -*- coding: utf-8 -*-
from pyzbus import Rpc  
rpc = Rpc(service = 'rpc')# host='172.24.180.27')

#user = {'name': u'中国', 'addr': 'address'}
print rpc.user('hong')

rpc.destroy()

